### ISSUE TITLE
Improvement Suggestions

### ISSUE THREAD

**Issue by ls-milkyway** [fifonik/FFMetrics#95]
This is a strange problem  ,,,,,,sometimes in the reference section with "path to file" ....the selected reference file changes to one of the distorted files & vice-versa ....specially when many distorted files & the reference file are in the same folder?? This occurs even when reference file is selected first or if distorted files are selected first??

Plus **results auto save files** can overwrite the previous file ...if it is not renamed. I lost my previous results once (6 hours of calculations)??
If possible then rectify please.

**Comment by fifonik** [comment]
Usually it is a good idea to create one ticket for one issue. Otherwise it might be hard to follow discussion.

> This is a strange problem ,,,,,,sometimes in the reference section with "path to file" ....the selected reference file changes to one of the distorted files & vice-versa

I have not seen such behaviour.
Are you sure that you are not drag & drop it manually? (this is possible).


> results auto save files can overwrite the previous file ...if it is not renamed.

I'll double check it later today, but if file exists the program should appent in it, not overwrite.

**Comment by fifonik** [comment]
I've checked code that saves results and file opened as:
`Stream = new StreamWriter(FS, true);`
So I do not know how results fil can be overwritten (assuming no issues in used NET)
